test/e2e/resource_quota.go

```diff
@@ -147,6 +148,84 @@ var _ = framework.KubeDescribe("ResourceQuota", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
+	It("should create a ResourceQuota and capture the life of an uninitialized pod.", func() {
```

Comment:
The test passes only with #50344. I can comment the test out in this PR, or squash this PR with #50344, up to the reviewers.

Reply:
I think it's fine if 50344 uncomments it.